libs/pdi-scanner/src/rust/client.rs

```diff
@@ -722,6 +733,26 @@ impl<T> Client<T> {
         )
     }
 
+    pub fn get_double_feed_detection_calibration_config(
+        &mut self,
+        timeout: Duration,
```

Comment:
We haven’t been super rigorous with our timeout values, but the way this function is implemented means that it could actually take up to `< timeout * 4` because each of the sub-calls is allowed to take up to `timeout`.

Reply:
Yeah, do you think that's an issue? Or just a little confusing?